### ISSUE TITLE
Windows 11 pear dev not working

### ISSUE THREAD

**Issue by connor-davis** [holepunchto/pear#182]
Hello, Excuse any improper knowledge as this is my first ever issue:

I have run into several issues with pear.

1. When running pear dev after stock pear init - I get 
![image](https://github.com/holepunchto/pear/assets/61504387/dd641126-ced1-4e67-99d4-f9c89ebf90b7)

2. Related to 1 - When I try again it gives 
![image](https://github.com/holepunchto/pear/assets/61504387/47a22425-034c-4eef-94aa-cb021d691a0e)

3. Another weird issue is that if I happen to have for example keet open, both of the above cases close/crash keet.

Pear Info:
![image](https://github.com/holepunchto/pear/assets/61504387/64c2ea13-f6ce-4cff-b4cb-81224294813c)

Windows Version:
![image](https://github.com/holepunchto/pear/assets/61504387/38b5ab72-e694-4a67-881f-e2c3f1dc4147)

**Comment by MrMasrozYTLIVE** [comment]
Can confirm on Win 10 Pro 22H2 (19045.2728

**Comment by MrMasrozYTLIVE** [comment]
Some additional information about Pear and NodeJS versions:
```
PS C:\> pear versions
Pear ~ Welcome to the Internet of Peers
pear: 0.3229.pqbzjhqyonxprx8hghxexnmctw75mr91ewqw5dxe1zmntfyaddqy

PS C:\> node --version
v20.5.1
```

**Comment by MrMasrozYTLIVE** [comment]
Somehow it fixed itself after running app from `C:/Users/<User>/Desktop/<AppFolder>` using `pear dev`
Running app from `E:/<AnotherPathHere>/<AppFolder>` using same command is not possible due to 2nd error from OP

**Comment by davidmarkclements** [comment]
interesting, @MrMasrozYTLIVE are C and E different types of disk (e.g. SSD vs spinning?). I'm assuming Pear is installed in your C:\ drive (see in [user]\AppData\Roaming\pear)?

@connor-davis are you always trying to run an app on a different drive to the one Pear is installed on?

**Comment by connor-davis** [comment]
I'm running my dev on my hard drive and pear is installed to the SSD yes... Is that what's going on here?

**Comment by MrMasrozYTLIVE** [comment]
> interesting, @MrMasrozYTLIVE are C and E different types of disk (e.g. SSD vs spinning)?. I'm assuming Pear is installed in your C:\ drive (see in [user]\AppData\Roaming\pear)?
Yes, it is installed on my SSD C:/ drive while project is on HDD E:/ drive. Running that same project from Desktop folder on C:/ works just fine while from E:/ it doesn't work and errors with ipc.start not defined error

**Comment by st10068305** [comment]
Yes, I just tried ```pear dev``` on the same drive as pear is installed and it runs without error...

**Comment by connor-davis** [comment]
Yes, I just tried ```pear dev``` on the same drive as pear is installed and it runs without error...

**Comment by MrMasrozYTLIVE** [comment]
> Yes, I just tried ```pear dev``` on the same drive as pear is installed and it runs without error...

Well i guess we found out the Issue. When running pears on different drive than its installed on - it doesn't work

**Comment by davidmarkclements** [comment]
@MrMasrozYTLIVE 50/50 on same drive or 50/50 on different drive?

**Comment by MrMasrozYTLIVE** [comment]
`C:\Users\<User>\Coding\NodeJS\LinkUp-P2P-Chat`: pear dev
```
x An error occured
TypeError: ipc.start is not a function
``` 

`C:\Users\<User>\Desktop\LinkUp-P2P-Chat`: pear dev
```
x An error occured
TypeError: ipc.start is not a function
```

But it did work yesterday from desktop. So i'd say its 50/50 at all. On second drive I never got it to run but on C:/ drive it did ran once

**Comment by MrMasrozYTLIVE** [comment]
Oh also it does run on linux devices just fine. So it happens only on windows devices, when I use the app on Arch Linux it works

**Comment by mafintosh** [comment]
fix for path issue here: https://github.com/holepunchto/pear/pull/193

**Comment by mafintosh** [comment]
of course!

**Comment by st10068305** [comment]
Shweet, Im looking forward!

**Comment by davidmarkclements** [comment]
should be resolved as of v0.4444.pqbzjhqyonxprx8hghxexnmctw75mr91ewqw5dxe1zmntfyaddqy (or before?) - closing
